### ISSUE TITLE
Required roles instead of requiredPermissions with @Command

### ISSUE THREAD

**Issue by xbony2** [BtoBastian/sdcf4j#38]
Or can roles translate to required permissions somehow? I dunno, the Javadocs are too vague for me.

**Comment by Vampire** [comment]
The permissions must be granted by your bot via `CommandHandler#addPermission`.
Currently you can only translate yourself by iterating through the roles and add them as permissions.

**Comment by xbony2** [comment]
So it's not anything to do Discord's permissions? >.> ugh, got confused with that. Thanks very much for your help.

**Comment by Vampire** [comment]
Nope, that is totally managed by your code only unless you translate discord perms to bot perms.
